packages/cli/commands/cms/reactModules.js

```diff
@@ -0,0 +1,80 @@
+const fs = require('fs');
+const { logger } = require('@hubspot/cli-lib/logger');
+const {
+  listGithubRepoContents,
+  downloadGithubRepoContents,
+} = require('@hubspot/local-dev-lib/github');
+const {
+  throwErrorWithMessage,
+} = require('@hubspot/local-dev-lib/errors/standardErrors');
+const { i18n } = require('../../lib/lang');
+const path = require('path');
+
+const i18nKey = 'cli.commands.cms.subcommands.reactModule';
+
+exports.command = 'get-react-module [name] [dest] [--list]';
+exports.describe = i18n(`${i18nKey}.describe`);
+
+exports.handler = async options => {
+  const { name, dest, list } = options;
+
+  //TODO: trackCommandUsage()
+  //TODO: i18n
+  //TODO: Move logic to local-dev-lib -- modules.ts -- fetchReactModules() and call from here
+
+  if (list) {
+    try {
+      const contents = await listGithubRepoContents(
+        'HubSpot/cms-sample-assets',
+        'modules/',
+        'dir'
+      );
+
+      logger.group('React modules available to download:');
+      contents.map(x => {
```

Comment:
not really a canonical use of `map`. Generally Map would be used to create a new array based on some transformation of they current array. Here you're just iterating on contents and logging - I think a `forEach` would make more sense.

Reply:
Yep - makes sense - Updated the code to reflect